.github/ISSUE_TEMPLATE/Standard Technical Issue.md

```diff
@@ -0,0 +1,19 @@
+## Description
```

Comment:
Can we change the name of this to not have spaces in it? spaces almost always end up causing a bunch of issues down the line. Also for the description on the issue, there' s a solution but nothing about what it's solving.

Reply:
renamed